package.json

```diff
@@ -30,7 +30,7 @@
     "jayschema": "^0.3.1",
     "js-yaml": "^3.2.7",
     "matrix-appservice": "^0.2.3",
-    "matrix-appservice-bridge": "^0.1.5",
+    "matrix-appservice-bridge": "matrix-org/matrix-appservice-bridge#develop",
```

Comment:
This is just temporary so Jenkins doesn't whine (since it `npm installs` from scratch). I don't want to constantly do releases each time I require a bug fix / new method to be added to the bridge lib atm.